lib/crypto.js

```diff
@@ -696,6 +697,71 @@ exports.setEngine = function setEngine(id, flags) {
   return binding.setEngine(id, flags);
 };
 
+const kMaxUint32 = Math.pow(2, 32) - 1;
+
+function randomFillSync(buf, offset = 0, size) {
+  if (!isUint8Array(buf)) {
+    throw new TypeError('"buf" argument must be a Buffer or Uint8Array');
+  }
+
+  if (typeof offset !== 'number' || offset < 0 || offset > kMaxUint32) {
+    throw new TypeError('offset must be a number >= 0');
```

Comment:
The error message for `offset > kMaxUint32` is kind of confusing (here and below.)